packages/transformers/svelte/src/SvelteTransformer.js

```diff
@@ -0,0 +1,147 @@
+import {Transformer} from '@parcel/plugin';
```

Comment:
This enable type checking with Flow for this file
```suggestion
// @flow
import {Transformer} from '@parcel/plugin';
```